src/test/schema_test.cpp

```diff
@@ -1318,3 +1318,222 @@ TEST(SchemaTest, CustomNodeConfigParamsInvalidType) {
+            }
+        ],
+        "pipeline_config_list": [
+            {
+                "name": "pipeline1Dummy",
+                "inputs": ["custom_dummy_input"],
+                "nodes": [
+                    {
+                        "name": "dummyNode",
+                        "library_name": "dummy_library",
+                        "type": "custom",
+                        "params": {
+                            "a": "1024",
+                            "b": "512"
+                        },
+                        "inputs": [
+                            {"b": {"node_name": "request",
+                                "data_item": "custom_dummy_input"}}
+                        ],
+                        "outputs": [
+                            {"data_item": "a",
+                            "alias": "new_dummy_output"}
+                        ],
+                        "demultiply_count": "10"
+                    }
+                ],
+                "outputs": [
+                    {"custom_dummy_output": {"node_name": "dummyNode",
+                                            "data_item": "new_dummy_output"}
+                    }
+                ]
+            }
+        ]
+    })";
+
+    rapidjson::Document demultiplexerConfigDemultiplyCountTypeInvalidParsed;
+    demultiplexerConfigDemultiplyCountTypeInvalidParsed.Parse(demultiplexerConfigDemultiplyCountTypeInvalid);
+    auto result = ovms::validateJsonAgainstSchema(demultiplexerConfigDemultiplyCountTypeInvalidParsed, ovms::MODELS_CONFIG_SCHEMA);
+    EXPECT_EQ(result, ovms::StatusCode::JSON_INVALID);
+}
+
+TEST(SchemaTest, DemultiplexerConfigDemultiplyCountNegative) {
+    const char* demultiplexerConfigDemultiplyCountNegative = R"(
+    {
+        "model_config_list": [
+            {
+                "config": {
+                    "name": "dummy",
+                    "base_path": "dummy_path",
+                    "target_device": "CPU",
+                    "model_version_policy": {"all": {}},
+                    "nireq": 1
+                }
+            }
+        ],
+        "pipeline_config_list": [
+            {
+                "name": "pipeline1Dummy",
+                "inputs": ["custom_dummy_input"],
+                "nodes": [
+                    {
+                        "name": "dummyNode",
+                        "library_name": "dummy_library",
+                        "type": "custom",
+                        "params": {
+                            "a": "1024",
+                            "b": "512"
+                        },
+                        "inputs": [
+                            {"b": {"node_name": "request",
+                                "data_item": "custom_dummy_input"}}
+                        ],
+                        "outputs": [
+                            {"data_item": "a",
+                            "alias": "new_dummy_output"}
+                        ],
+                        "demultiply_count": -1
```

Comment:
Maybe it would be better to have 1 static demultiplexerConfigDemultiplyCount, and then replace in tests the value with different parameters? 

Additionally we should have test for demutliply = 0,1 documenting behavior for those ones.